src/components/colorPicker/ColorPickerDialog.js

```diff
@@ -294,11 +295,11 @@ class ColorPickerDialog extends PureComponent {
   }
 
   renderDialog() {
-    const {visible, dialogProps, testID} = this.props;
+    const {dialogProps, testID} = this.props;
+    const dialogProp = extractComponentProps(Dialog, this.props);
```

Comment:
What's the reason for that? 
Why would you want to deprecate a dedicated prop for dialog? 

I don't think flatting dialog props into ColorPicker props is a good call. 
ColorPicker doesn't really inherit a Dialog, it is more compose of a Dialog.

Reply:
ColorPicker doesn't inherit Dialog but ColorPickerDialog is kinda is and 'dialogProps' prop is its prop to deprecate...

Reply:
but we still want to have a `dialogProps` prop so we can control how the dialog of ColorPicker looks.. no?

Reply:
I don't think we need a prop for that. The user should pass Dialog's props, not inside the 'dialogProps' object

Reply:
Ok, I got a little confused...
I see you want to remove `dialogProps` from `ColorPickerDialog` - that makes sense!
But we should still have a `dialogProps` prop in the main ColorPicker component for channeling these prop to `ColorPickerDialog`.

In the previous refactor you spread `ColorPickerDialog` propsTypes into ColorPicker so it was included. 
Now, removing it will remove it completely - you should declare it in the main component. 

Let's talk if you still think otherwise, ill explain in details why do we need it..

Reply:
I want the user to pass Dialog props to ColorPicker as he would to the Dialog and I will extract these props and pass them to ColorPickerDialog. I don't think there is a need for the object wrapper that is 'dialogProps' prop. What do you think?

Reply:
I think that there's a need for it. 
Let's say you have a prop called `containerStyle` in both Dialog and ColorPicker component. 
It can creates conflicts. 
I believe the separation here is important and having a dedicate prop as `dialogProps` is crucial. 

I think the thumb rule in this case is whether the internal component is also a standalone component (in this case, dialog is also a standalone component) 

If we'll take another case, for instance ListItem and ListItemLeftPart. 
ListItemLeftPart is just an internal implementation of ListItem, it doesn't live on its own, therefore it make sense to spread ListItemLeftPart props in ListItem props.

Does it make it sense?

Reply:
Ok. I'm closing this PR